src/purchase_listinator/models/internal/shopping_cart.clj

```diff
@@ -11,13 +11,15 @@
    :event-type   (s/eq :reorder-category)
    :shopping-id  s/Uuid
    :category-id  s/Uuid
+   :old-position s/Int
    :new-position s/Int})
 
 (s/defschema ReorderItemEvent
   {:moment          s/Num
    :event-type      (s/eq :reorder-item)
    :shopping-id     s/Uuid
    :item-id         s/Uuid
+   :old-position    s/Int
```

Comment:
We can't trust old positions coming from the customer, because they might have an old version of the shopping list.

I made a mistake keeping the old-position in the wire :(

Same for Categories

Reply:
I agree. I am going to revert the unnecessary changes.